### DISCUSSION TITLE
How to change the default movement speed ?

### DISCUSSION THREAD

**Discussion by Walach76** [Rughalt/D35E#772] [Q&A]
Hello, 
I'm French and I would like to customise the default value for the movement speed of the actors, as I use the metric system measurement and not the imperial one, all my measures are wrong by default. I currently have to edit manually every character sheet. But, some values are also calculated. For example, 30 feet is 9 meters, but when a character has 9 meters and I assign him some race (dwarf for example), the calculated value (-1 = 9-10) appears...
Is there a way (by editing a config file for example) to modify this default value (I found where of the maths are done for racial changes...) but can't find where this damn 30 is put in the first place. 

Thanks guys !

**Comment by Walach76** [comment]
Well, looks like I found what I was searching for, it's in "template.json" file in the root directory of the module. I will try to change this value and see what happens :)

**Comment by Rughalt** [comment]
Currently, the metric system is not supported and there is an ongoing discussion on how to support it. Please note that changes like that will be overridden with system updates and there are places in the system that will still use feet.
